app/charts/shared/colors.tsx

```diff
@@ -5,3 +5,16 @@ export const convertHexToRgbArray = (hex: string): [number, number, number] => {
 
   return [r, g, b];
 };
+
+export const convertRgbArrayToHex = (rgbArray: number[]): string => {
+  switch (rgbArray.length) {
+    case 3:
+      return `rgb(${rgbArray.join(",")})`;
+    case 4:
+      return `rgba(${rgbArray.join(",")})`;
+    default:
+      throw new Error(
+        `You need to pass 3 or 4 arguments when converting RGB array to HEX, while ${rgbArray.length} were provided.`
+      );
+  }
+};
```

Comment:
Should be fixed up into the first commit of this PR.

Reply:
Good point, done!